src/MathExpression/ShuntingYard.java

```diff
@@ -0,0 +1,76 @@
+package MathExpression;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+import MathExpression.Token.Type;
+
+public class ShuntingYard {
+
+    private static void processToken(Token token, List<Token> postfix, Stack<Token> shunt) {
+        switch(token.type) {
```

Comment:
It is not needed but i find the following style to write such switch cases in this scenario a little bit more readable:
```suggestion
       switch (token.type) {
            case Number, Parameter -> {
                postfix.add(token);
            }
```
and so forth... But thats for you to decide whichever you like best.

Reply:
In the case of the large switch in ShuntingYard, it is more readable the way it is, I think

Reply:
Thats ok ^^